pyanaconda/install.py

```diff
@@ -131,6 +134,10 @@ def doConfiguration(storage, payload, ksdata, instClass):
     # write out the user interaction config file
     screen_access.sam.write_out_config_file()
 
+    if ksdata.snapshot:
+        with progress_report(N_("Creating snapshots")):
+            ksdata.snapshot.execute(storage, ksdata, instClass)
+
```

Comment:
True, fixed.
I've created new checking method for this which is used also before calling `setup` and `execute` for pre snapshots.